test/plots/piePlotTests.ts

```diff
@@ -264,6 +264,25 @@ describe("Plots", () => {
         });
         svg.remove();
       });
+
+      it("labels outside of the render area are hidden", () => {
+        piePlot.detach();
+        piePlot.removeDataset(simpleDataset);
+        var data = [5000, 5000, 5000];
```

Comment:
Can you explain this test? For the given data it does not make any sense that only one label is hidden.